.pre-commit-config.yaml

```diff
@@ -60,7 +60,7 @@ repos:
           - tomli
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.4.7"
+    rev: "v0.5.0"
     hooks:
       - id: ruff
         args: ["--fix", "--show-fixes"]
```

Comment:
[Have you checked if we still need `--show-fixes`?](https://astral.sh/blog/ruff-v0.5.0#output-format-now-defaults-to-full)

Reply:
We do. Here's a screenshot of ruff autofixing the same lint twice, first with then without this flag
![Screenshot 2024-06-28 at 21 57 06](https://github.com/astropy/astropy/assets/14075922/6e20a271-25ef-40ff-a3b2-0c5ffe525904)